docs/api/resource.rst

```diff
@@ -192,9 +194,9 @@ HTTP Status Codes
 -----------------
 
 * ``200 OK``: The request was processed
-* ``304 Not Modified``: Collection did not change since value in ``If-Modified-Since`` header
+* ``304 Not Modified``: Collection did not change since value in ``If-None-Match`` header
 * ``400 Bad Request``: The request querystring is invalid
-* ``412 Precondition Failed``: Collection changed since value in ``If-Unmodified-Since`` header
```

Comment:
Yes, that's how we did it. In brief:
* 304 on read operations when client version is «up to date»
* 412 on write operations when server version has changed meanwhile